src/main/resources/requests/accountManagementRequests.ts

```diff
@@ -162,6 +163,22 @@ export class AccountManagementRequests {
     }
   }
 
+  public async getPiUserByCftID(oid: string): Promise<any> {
+    try {
+      const response = await accountManagementApi.get(`/account/provenance/CFT_IDAM/${oid}`);
+      return response.data;
+    } catch (error) {
+      if (error.response) {
+        logger.error('Failed to GET PI user request', error.response.data);
+      } else if (error.request) {
+        logger.error('Request failed for Pi user', error.request);
+      } else {
+        logger.error('Something went wrong trying to get the pi user from the oid', error.message);
```

Comment:
The error message says 'oid' here. I presume that is PI_AAD specific?

Reply:
Updated :)